### PR TITLE
Comment box

### PR DESCRIPTION
**Pull request by tonianner**
Users can now comment
Comments appear only in the restaurant ID
Dummy seeds added

### CODE REVIEW

File: app/assets/javascripts/comments.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```
Comment (dshcheung) [inline app/assets/javascripts/comments.coffee]: remove if not needed

File: app/assets/stylesheets/comments.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the comments controller here.
+// They will automatically be included in application.css.
```
Comment (dshcheung) [inline app/assets/stylesheets/comments.scss]: please remove

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      redirect_to restaurant_path(params[:restaurant_id])
+    else
+      redirect_to restaurant_path(params[:restaurant_id])
+    end
+  end
+end
+
+
+
+# /restuarant/:restaurant_id/comments
```
Comment (dshcheung) [inline app/controllers/comments_controller.rb]: please remove this useless commendation and remove these extra lines too

File: app/controllers/restaurants_controller.rb [hunk 4]
```diff
@@ -12,5 +12,9 @@ def search
 
   def show
     @restaurant = Restaurant.find(params[:id])
+    @comments = @restaurant.comments
+    @comment = Comment.new
   end
 end
+
+
```
Comment (dshcheung) [inline app/controllers/restaurants_controller.rb]: you only need 1 extra line, not 2

File: app/controllers/sessions_controller.rb [hunk 5]
```diff
@@ -20,3 +20,5 @@ def destroy
     redirect_to root_url, notice: "logged out!"
   end
 end
+
+
```
Comment (dshcheung) [inline app/controllers/sessions_controller.rb]: you only need 1 extra line, not 2

File: app/views/comments/create.html.erb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<h1>Comments#create</h1>
+<p>Find me in app/views/comments/create.html.erb</p>
```
Comment (dshcheung) [inline app/views/comments/create.html.erb]: please remove this if not needed

File: app/views/restaurants/show.html.erb [hunk 7]
```diff
@@ -15,3 +15,19 @@
...
+    </div>
+      <%= submit_tag 'Submit', class:"btn btn-primary" %>
+  <% end %>
+</div>
+
+<div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-8 col-md-offset-2 well">
+  <%= render 'comments/box', comments: @comments %>
+</div>
+
+
```
Comment (dshcheung) [inline app/views/restaurants/show.html.erb]: remove extra lines. you only need 1 extra line

File: app/views/sessions/new.html.erb [hunk 8]
```diff
@@ -17,7 +17,7 @@
         </div>
         <br>
         <div class="actions">
-          <button class="btn btn-primary"> Log in </button>
+          <%= submit_tag 'Log in', class:"btn btn-primary" %>
         </div>
     <% end %>
```
Comment (dshcheung) [inline app/views/sessions/new.html.erb:26]: indentations

**Comment by dshcheung** [comment]
rebase and review comments please

**Comment by cryptokoala4** [comment]
fixed, but there's still conflicts after i rebased

**Comment by cryptokoala4** [comment]
merge conflict fixed by fer, looking for a merge request

**Comment by dshcheung** [comment]
can merge after amend

**Pull request merged**